### PR TITLE
Example code for Animating on custom events (#4635)

### PR DESCRIPTION
**Pull request by kewalkishang**
**Description:**
Need for example regarding the use of custom events to trigger animations in A-Frame. The current documentation does not explicitly show how to use the .emit() function to start animations based on custom events, which can be confusing for new users.

**Changes proposed:**
- Created a glitch example that demonstrates how to trigger animations using startEvents, pauseEvents, resumeEvents for custom events and then emitting them.
- Added a link to the glitch example in the animation doc.

### CODE REVIEW

**Comment by dmarcos** [comment]
Thanks. The example uses an old version of A-Frame (1.6.0). Also I don't see any animations being triggered. I might be missing something

**Comment by kewalkishang** [comment]
Hi @dmarcos, thanks for the review. 
Updated A-Frame to the latest version (Had made the example a while back).
The animation is on the cube. It changes its color from Red to blue in 2.5 seconds now(It was 5 seconds before). The idea was to give users ample time to pause the animation in between and try the pauseEvents and resumeEvents as well.
Do let me know how I can make the example even better.

Best, 
Kewal

**Comment by dmarcos** [comment]
Thanks for the patience. I'm reluctant to add more glitches to the examples because those need maintenance too and easily got out of date. Could we handle this by adding a couple of lines and pointing to the custom events section of the docs? 

https://aframe.io/docs/1.6.0/introduction/javascript-events-dom-apis.html#emitting-an-event-with-emit

Thanks

**Comment by kewalkishang** [comment]
Hey @dmarcos.
I see. Makes sense.
Added a simple custom event in the existing example and a link to the custom events section for reference.
[Changes](https://github.com/kewalkishang/aframe/blob/event-driven-animation-example/docs/components/animation.md#animating-on-events)
Do let me know if this is what you had in mind.

Thanks.

File: docs/components/animation.md [hunk 1]
```diff
@@ -140,15 +140,16 @@ We can use the `startEvents` property to animate upon events:
   geometry="primitive: box"
   material="color: red"
   animation__mouseenter="property: components.material.material.color; type: color; to: blue; startEvents: mouseenter; dur: 500";
-  animation__mouseleave="property: components.material.material.color; type: color; to: red; startEvents: mouseleave; dur: 500";>
+  animation__mouseleave="property: components.material.material.color; type: color; to: red; startEvents: mouseleave; dur: 500";
+  animation__customevent="property: components.material.material.color; type: color; from: red; to: blue; startEvents: startanim001; dur: 500";>
```
Comment (dmarcos) [inline docs/components/animation.md:144]: can we change the event name? startanim001 not super friendly. What about 'triggeranimation`?

**Comment by dmarcos** [comment]
Thanks so much for sticking with it!

**Pull request merged**